tailwind.css

```diff
@@ -71,5 +72,6 @@
     --blue-300: 177 202 236;
     --blue-200: 216 228 245;
     --blue-100: 61 122 207;
+    --blue-50: 239 244 251; /** replace with correct dark color **/
```

Comment:
still pending ref: https://www.figma.com/file/Bksz1JeDYxQVIXdI46EgPT?node-id=835:50395&mode=dev#691690703 cc/ @erikalogie

Reply:
replied in Figma